Edit entries swiftui terms syntax (#1717)
Codecademy/docs: content/swiftui/concepts/viewmodifier/terms/resizable/resizable.md

  - 'paths/build-ios-apps-with-swiftui'
---

> The **`.resizable()`** modifier method configures an image in a [`View`](https://www.codecademy.com/resources/docs/swiftui/views) to resize itself to fit the surrounding space of its parent container.

Reviewer: ```suggestion
The **`.resizable()`** modifier method configures an image in a [`View`](https://www.codecademy.com/resources/docs/swiftui/views), resizing itself to fit the surrounding space of its parent container.
```

## Syntax

```pseudo
Image("image-file")
    .resizable()
```

> The `.resizable()` modifier is applied to the [`Image`](https://www.codecademy.com/resources/docs/swiftui/views/image) view, which makes the image resize itself to fit its space.

Reviewer: ```suggestion
The `.resizable()` modifier is applied to an [image](https://www.codecademy.com/resources/docs/swiftui/views/image), which makes the image resize itself to fit its space.
```

Author: I deleted "accepts" from the sentence.

> ### Optional Parameters

Reviewer: ```suggestion
```

> The `.resizable()` modifier accepts two optional parameters:
> 

Reviewer: ```suggestion
```

Reviewer: Incorporated into the paragraph above.

Author: Yes, that looks good.

## Example (Without Parameters)

> The following example creates an image of a red circle that is resizable and fills the space of its ['VStack'](https://www.codecademy.com/resources/docs/swiftui/views/vstack) parent view, which has a size of `300x300` points, and a border to show its size:

Reviewer: ```suggestion
The following example creates an image of a red circle that is resizable and fills the space of its ['VStack'](https://www.codecademy.com/resources/docs/swiftui/views/vstack) parent view, which has a size of `300` by `300` points (`pt`), and a border to show its size:
```

}
```

The left image shows the red circle without the `.resizable()` modifier not filling the available space. The right image shows the red circle with the `.resizable()` modifier filling up the space:

![SwiftUI ViewModifier .resizable()](https://raw.githubusercontent.com/Codecademy/docs/main/media/swiftui-resizable.png)

## Example (With Parameters)

> The following example sets the `capInsets: EdgeInsets()` for `top`, `leading`, `bottom`, and `trailing` edges to `20`. Next, the image is tiled within the parent container by setting the `resizingMode: .tile`. This tiles many red circles within the `300x300` parent container and sets the space between the circles to `20`, making them closer together. The circles would be further apart if a lower value of `10` is used. If `50` is used, the circles would push together, forming a giant red square.

Reviewer: ```suggestion
The following example sets the `capInsets: EdgeInsets()` for `top`, `leading`, `bottom`, and `trailing` edges to `20`. Next, the image is tiled within the parent container by setting the `resizingMode: .tile`. This tiles many red circles within the `300` by `300` parent container and sets the space between the circles to `20`, making them closer together. The circles would be further apart if a lower value of `10` is used. If `50` is used, the circles would push together, forming a giant red square.
```
